src/Functions/degrees.cpp

```diff
@@ -0,0 +1,27 @@
+#include <Functions/FunctionFactory.h>
+#include <Functions/FunctionMathUnary.h>
+
+template <typename T>
+T degrees(T r)
```

Comment:
Put it into anonymous namespace or make the function static.

Reply:
sure moved into the anonymous namespace.